src/omnibar/omnibar.ts

```diff
@@ -198,11 +208,26 @@ body {
   line-height: 24px;
   overflow: hidden;
   padding: 0 15px;
-  text-align: right;
+  position: relative;
   text-overflow: ellipsis;
   white-space: nowrap;
 }
 
+.sky-omnibar-environment-info {
```

Comment:
I would look into using flexbox or another technique rather than `float: right`, since floating elements has an effect on subsequent elements and usually needs a `clear: both` after it.

Reply:
Per our side conversation, I changed these new elements to `span`s.